packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -58,6 +62,14 @@ export class SCWSigner implements Signer {
     this.request = this.request.bind(this);
     this.createRequestMessage = this.createRequestMessage.bind(this);
     this.decryptResponseMessage = this.decryptResponseMessage.bind(this);
+
+    // rehydrate the sub account store
+    SubAccount.persist.rehydrate(); // should this be called inside the createCoinbaseWalletSDK?
```

Comment:
What is this for?

Reply:
Hydrates the sub account store with localstorage data if present.